### PR TITLE
Added support for additional languages (cn, it, ru, pt)

### LINKED ISSUE #16: Other Languages ?
Hey @psistorm what if i need other languages?
What do i have to change where, if one need other languages?

### PR DESCRIPTION
**Pull request by lorenzschmid**
Added all remaining languages supported by LEO to the workflow (fixes #16)
Changed the flags to have a consistent look among all languages.

### CODE REVIEW

**Comment by kulturbande** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -7,14 +7,18 @@ Leo Dictionary
 ---------
 ![Searching with leo](Leo%20Dictionary/screenshot_de.png)
 
-Searches the Leo dictionary (http://dict.leo.org) for the languages german, english, french, spanish and polish. Feeds back the results into Alfred for instantly seeing the translated word. When hitting 'Enter' the translated word is copied to the clipboard. When hitting 'Shift + Enter' the selected word translation is opened with Leo.
+Searches the Leo dictionary (http://dict.leo.org) for the languages German, English, French, Spanish and Polish. Feeds back the results into Alfred for instantly seeing the translated word. When hitting 'Enter' the translated word is copied to the clipboard. When hitting 'Shift + Enter' the selected word translation is opened with Leo.
 With Version 1.2 newly introduced a somewhat "automatic" detection of the language of the entered word. 
 
 There are three ways to search Leo:
```
Comment (kulturbande) [inline README.md]: There are now eight ways to search ;-)

**Comment by winkelsdorf** [comment]
Great work, thank you 👍 

+1 for a merge.

**Comment by psistorm** [comment]
Sorry for the late merge :)

**Pull request merged**